index.bs

```diff
@@ -1418,7 +1417,14 @@ is much more readable than
 new Event("example", true, false)
 </pre>
 
-Also,
+</div>
+
+API designers should also consider
```

Comment:
You should use active tone (following the style guide) here - e.g. You should consider accepting parameters through a dictionary.